akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorContextImpl.scala

```diff
@@ -62,6 +67,16 @@ import scala.concurrent.ExecutionContextExecutor
   override def spawnAdapter[U](f: java.util.function.Function[U, T], name: String): akka.actor.typed.ActorRef[U] =
     internalSpawnAdapter(f.apply, name)
 
+  // FIXME missing implicit error message
+  override def ask[Req, Res](otherActor: ActorRef[Req], createMessage: ActorRef[Res] ⇒ Req)(responseToOwnProtocol: Try[Res] ⇒ T)(implicit responseTimeout: Timeout, classTag: ClassTag[Res]): Unit = {
+    import akka.actor.typed.scaladsl.AskPattern._
+    implicit val scheduler = system.scheduler
+
+    (new Askable(otherActor) ? createMessage).onComplete(res ⇒
```

Comment:
is `Askable` needed? done by `import AskPattern._`?

Reply:
I got a conflict because of adding our own implicitries here with `?` , but if I remove that everything is fine.